### ISSUE TITLE
Training loss problem

### ISSUE THREAD

**Issue by COST-97** [hpcaitech/Open-Sora#402]
Hello:
I used some new data to fine-tune on opensora1.1, but the noise loss can only drop from about 1 to about 0.2, and cannot continue to decrease. And after training for a long enough time, the loss increases instead. Did your model encounter similar problems during the training process?

**Comment by qian18long** [comment]
Hi, I have similar problem. How did you solve this problem?
